spec/models/status_spec.rb

```diff
@@ -120,7 +120,57 @@
   end
 
   describe '#permitted?' do
-    pending
+    it 'returns true when direct and account is viewer' do
+      status = Fabricate(:status, visibility: :direct)
+      expect(status.permitted?(status.account)).to be true
+    end
+
+    it 'returns true when direct and viewer is mentioned' do
+      viewer = Fabricate(:account)
```

Comment:
Can you use [`alice`](https://github.com/tootsuite/mastodon/pull/2982/files#diff-c479d72b97713a99cab1561b1a8ee212L4)?